### PR TITLE
MOON-220: Avoid to render something when the component is empty

### PR DESCRIPTION
**Pull request by ffffffelix**
## JIRA

https://jira.jahia.org/browse/MOON-220

## Description

Avoid rendering something when the component is empty and have no children/items.

## Tests

The following are included in this PR

- [X] Unit Test

### CODE REVIEW

**Comment by ffffffelix** [comment]
Some warnings are displayed during tests because I have to don't pass some required props to verify if the component displays nothing when it has no children.
Does someone have an idea to prevent that?

File: src/components/Badge/Badge.tsx [hunk 1]
```diff
@@ -51,13 +51,19 @@ export const Badge: FunctionComponent<BadgeProps> = ({
         `moonstone-${type}`,
         className
     );
+
+    if (!label || label.length < 1) {
```
Comment (vindhya) [inline src/components/Badge/Badge.tsx:55]: Can't the diamond type of the Badge be used without a label?

Comment (ffffffelix) [reply src/components/Badge/Badge.tsx:55]: You right!
I'm wondering if this variant is used, I'd like to remove it.

Comment (vindhya) [reply src/components/Badge/Badge.tsx]: I haven't seen it being used: https://github.com/search?q=org%3AJahia+type%3D%22diamond%22&type=code

**Review by vindhya — APPROVED** [review]

**Pull request merged**